### ISSUE TITLE
This module is declared with 'export =', and can only be used with a default import when using the 'esModuleInterop' flag

### ISSUE THREAD

**Issue by robertd** [trapcodeio/cron-time#16]
```ts
error TS1259: Module '"/Users/rdjurasaj/code/ngtoc-devops/data-transfer-factory/node_modules/cron-time-generator/dist/index"' can only be default-imported using the 'esModuleInterop' flag

4 import cronTime from "cron-time-generator";
         ~~~~~~~~

  node_modules/cron-time-generator/dist/index.d.ts:203:1
    203 export = CronTime;
        ~~~~~~~~~~~~~~~~~~
    This module is declared with 'export =', and can only be used with a default import when using the 'esModuleInterop' flag.
```

It works with `esModuleInterop` flag in `tsconfig.json` but it's a workaround.

**Comment by trapcodeio** [comment]
The latest version will fix this, version `2.0.0`
Try it out and let me know

https://github.com/trapcodeio/cron-time/releases/tag/v2.0.0

**Comment by robertd** [comment]
That did the trick. Thanks @trapcodeio 👍
